torchtune/modules/model_fusion/_fusion_utils.py

```diff
@@ -37,3 +37,33 @@ def fusion_params(self) -> List[str]:
         return [k for k, v in self.named_parameters()]
 
     module.fusion_params = functools.partial(fusion_params, module)
+
+
+def get_fusion_params(model: nn.Module) -> Dict[str, nn.Parameter]:
```

Comment:
What's the purpose of this function? Why would you need it?

Reply:
It allows you to control which parameters you want to freeze. It's similar to get_peft_params. For an example of this, look in the DeepFusionModel __init__ for an example.